datafusion/proto/src/bytes/registry.rs

```diff
@@ -42,4 +42,16 @@ impl FunctionRegistry for NoRegistry {
     fn udwf(&self, name: &str) -> Result<Arc<WindowUDF>> {
         plan_err!("No function registry provided to deserialize, so can not deserialize User Defined Window Function '{name}'")
     }
+    fn register_udaf(
+        &mut self,
+        _udaf: Arc<AggregateUDF>,
+    ) -> Result<Option<Arc<AggregateUDF>>> {
+        unimplemented!()
```

Comment:
Likewise can you please change this to `plan_err` to mirror the function calls above?